dispatchr-ruby/config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  match '/users/login/' => 'users#login', :via => :get
+  # match '/users/login/' => 'users#login', :via => :get
   match '/requests/user/' => 'requests#search_user', :via => :get
   match '/request_items/request/' => 'request_items#search_request', :via => :get
   resources :items, :requests, :users, :request_items, :addresses
+  post '/users' => 'users#create'
+  post '/login' => 'sessions#create'
+  delete '/logout' => 'sessions#destroy'
+  post '/sign_up' => 'registrations#create'
+  match '/requests/accept/' => 'requests#accept_request', :via => :post
```

Comment:
Lets make this look like this
```ruby
post '/requests/accept/:id', to: 'requests#accept_request'
```
That way i can request with the url like this "http://localhost:3000/requests/accept/1" and not have to stick the id in the post json data.